packages/sdks/snippets/react/src/routes/custom-components/custom-child.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * Quickstart snippet
+ * snippets/react/routes/custom-child.tsx
+ */
+
+import {
+    Content,
+    fetchOneEntry,
+    getBuilderSearchParams,
+    isPreviewing,
+    type BuilderContent,
+  } from '@builder.io/sdk-react';
+  import { useEffect, useState } from 'react';
+import CustomHero from '../../components/CustomHero';
+  
+  const BUILDER_API_KEY = 'ee9f13b4981e489a9a1209887695ef2b';
+  const MODEL_NAME = 'page';
+  
+  // set whether you're using the Visual Editor,
+  // whether there are changes,
+  // and render the content if found
```

Comment:
I find this comment a bit confusing, especially "set whether you're using the Visual Editor". You're not actually setting that, you're just getting the value from `isPreviewing`. 

I would get rid of it completely: the comments throughout the component offer enough clarity IMO.